### PR TITLE
Update context around NFS for OCP 3.9.z

### PR DESCRIPTION
**Pull request by bmcelvee**

### CODE REVIEW

**Review by sdodson — APPROVED** [review]
/lgtm

**Comment by bmcelvee** [comment]
Thanks! @openshift/team-documentation PTAL :sunflower:

File: install_config/install/advanced_install.adoc [hunk 1]
```diff
@@ -984,6 +984,37 @@ openshift_hosted_registry_storage_volume_name=registry
...
 
+[discrete]
+[[advanced-install-cluster-registry-storage-upgrade-nfs]]
+===== Upgrading or Installing {product-title} with NFS
+
+The use of NFS for the core {product-title} components is not recommended, as NFS
+(and the NFS Protocol) does not provide the proper consistency needed for the
+applications that make up the {product-title} infrastructure.
+
+As a result, the installer/update playbooks require an option to enable the use
```
Comment (kalexand-rh) [inline install_config/install/advanced_install.adoc]: s/installer/update/installer and update

File: install_config/install/advanced_install.adoc [hunk 2]
```diff
@@ -984,6 +984,37 @@ openshift_hosted_registry_storage_volume_name=registry
...
+
+If you see the following messages when upgrading or installing an {product-title}
+3.9.z cluster, then an additional step is required.
+
+----
+TASK [Run variable sanity checks] **********************************************
+fatal: [host.example.com]: FAILED! => {"failed": true, "msg": "last_checked_host: host.example.com, last_checked_var: openshift_hosted_registry_storage_kind;nfs is an unsupported type for openshift_hosted_registry_storage_kind. openshift_enable_unsupported_configurations=True mustbe specified to continue with this configuration."}
+----
+
+In your Ansible inventory file, specify the following:
```
Comment (kalexand-rh) [inline install_config/install/advanced_install.adoc]: s/following:/following parameter:

**Review by kalexand-rh — COMMENTED** [review]
A couple of picks that apply to all three sections. Otherwise looks good!

**Comment by bmcelvee** [comment]
Thanks, @kalexand-rh!

**Pull request merged**